mercury/mercury-library/protocol-issue-credential/src/main/scala/io/iohk/atala/mercury/protocol/issuecredential/IssueFormats.scala

```diff
@@ -0,0 +1,153 @@
+package io.iohk.atala.mercury.protocol.issuecredential
+
+import io.circe.generic.semiauto.*
+import io.circe.{Decoder, Encoder}
+
+/*
+Issue Credential Formats:
+  Propose  ->  Offer -> Request  -> Issue
+
+ - Issue Propose:
+   - (DID Comm v1) https://github.com/hyperledger/aries-rfcs/tree/main/features/0453-issue-credential-v2#propose-attachment-registry
+     - dif/credential-manifest@v1.0
+     - aries/ld-proof-vc-detail@v1.0
+     - hlindy/cred-filter@v2.0
+     - anoncreds/credential-filter@v1.0
+   - (DID Comm v2) https://github.com/decentralized-identity/waci-didcomm/blob/main/issue_credential/README.md#propose-attachment-registry
+     - dif/credential-manifest@v1.0
+     - aries/ld-proof-vc-detail@v1.0
+     - hlindy/cred-filter@v2.0
+
+ - Issue Offer:
+   - (DID Comm v1) https://github.com/hyperledger/aries-rfcs/tree/main/features/0453-issue-credential-v2#offer-attachment-registry
+     - dif/credential-manifest@v1.0
+     - hlindy/cred-abstract@v2.0
+     - aries/ld-proof-vc-detail@v1.0
+     - anoncreds/credential-offer@v1.0
+   - (DID Comm v2) https://github.com/decentralized-identity/waci-didcomm/blob/main/issue_credential/README.md#offer-attachment-registry
+     - dif/credential-manifest@v1.0
+     - hlindy/cred-req@v2.0
+     - aries/ld-proof-vc-detail@v1.0
+
+ - Issue Request:
+   - (DID Comm v1) https://github.com/hyperledger/aries-rfcs/tree/main/features/0453-issue-credential-v2#request-attachment-registry
+     - dif/credential-manifest@v1.0
+     - hlindy/cred-req@v2.0
+     - aries/ld-proof-vc-detail@v1.0
+     - anoncreds/credential-request@v1.0
+   - (DID Comm v2) https://github.com/decentralized-identity/waci-didcomm/blob/main/issue_credential/README.md#request-attachment-registry
+     - dif/credential-manifest@v1.0
+     - hlindy/cred-req@v2.0
+     - aries/ld-proof-vc-detail@v1.0
+
+ - Issue Credential:
+   - (DID Comm v1) https://github.com/hyperledger/aries-rfcs/tree/main/features/0453-issue-credential-v2#credentials-attachment-registry
+     - aries/ld-proof-vc@v1.0
+     - hlindy/cred@v2.0
+     - anoncreds/credential@v1.0
+   - (DID Comm v2) https://github.com/decentralized-identity/waci-didcomm/blob/main/issue_credential/README.md#credentials-attachment-registry
+     - aries/ld-proof-vc@v1.0
+     - hlindy/cred@v2.0
+ */
+
+/** Issue Propose:
+  * @see
+  *   https://github.com/hyperledger/aries-rfcs/tree/main/features/0453-issue-credential-v2#propose-attachment-registry
+  *   (DID Comm v1)
+  *   - dif/credential-manifest@v1.0
+  *   - aries/ld-proof-vc-detail@v1.0
+  *   - hlindy/cred-filter@v2.0
+  *   - anoncreds/credential-filter@v1.0
+  * @see
+  *   https://github.com/decentralized-identity/waci-didcomm/blob/main/issue_credential/README.md#propose-attachment-registry
+  *   (DID Comm v2)
+  *   - dif/credential-manifest@v1.0
+  *   - aries/ld-proof-vc-detail@v1.0
+  *   - hlindy/cred-filter@v2.0
```

Comment:
Are all these formats similar?

Reply:
The intent is similar, and the holder uses it to specify characteristics of the VC during the `propose` phase of the issue flow. Each format is specific to a standardisation body (DIF, W3C, HL). By the way, we have not implemented this phase yet.